.github/ISSUE_TEMPLATE/service_request_template.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Service Request Template
+about: Template for requests or changes to current backend services or assistance with troubleshooting
+title: 'Service Request'
+assignees: ''
+labels: support-request
+---
+
+Instructions: Replace text below with details corresponding to your issue
+
+**TEAM NAME:** Enter Name  
+**PRIORITY (1-5):** Enter your priority  
+
+PRIORITY KEY - DELETE BEFORE SAVING ISSUE
+
+P1: This is a production outage and needs attention ASAP  
+P2: This is preventing development  
+P3: This needs to be fixed but development isn't blocked  
+P4: I need assistance at some point on an active project  
+P5: This is an idea that doesn't pertain to an active project  
+
+**DO NOT INCLUDE ANY SECRETS IN THIS REQUEST. IT IS PUBLICLY ACCESSIBLE**
+
+**Description of issue**
+
+> Describe the issue or need you are experiencing
+
+**Error Message/Logs**
+
+> Can you provide the error message or other logs which indicate the error. Examples: Browser's Javascript Console, AWS CloudWatch logs, terminal output. Please provide in code blocks
+
+**Recreation Steps**
```

Comment:
In my universe the overwhelmingly more common phrase is "Reproduction Steps" - I recommend we not confuse those who are familiar with the repro terminology